### DISCUSSION TITLE
Environment variables in dev and prod

### DISCUSSION THREAD

**Discussion by kye79** [vercel/next.js#12893] [Help]
My app seems to be able to access environment variables just fine when I start the app with `npm run dev`, but it doesn't have access to it when I run `npm run start`.

I set the env vars using `export X=Y` on Linux

Any idea why? Am I missing a step?

**Comment by leerob** [comment]
Would you mind sharing an example or some code? 

Maybe try using a `.env` file to store your [environment variables](https://nextjs.org/docs/basic-features/environment-variables).
